src/PresentationalComponents/Cards/OverviewDashbarCard/components/DashbarCardTagOrIcon/DashbarCardTagOrIcon.cy.js

```diff
@@ -0,0 +1,97 @@
+import React from "react";
+import { mount } from "cypress/react";
+
+import { DashbarCardTagOrIcon } from "./DashbarCardTagOrIcon";
+import { IntlProvider } from "@redhat-cloud-services/frontend-components-translations/index";
+import messages from "../../../../../../locales/translations.json";
+import tooltipMessages from "../../../../../Messages";
+import {
+  PATHWAYS,
+  INCIDENTS,
+  IMPORTANT_RECOMMENDATIONS,
+  CRITICAL_RECOMMENDATIONS,
+} from "../../../../../AppConstants";
+
+// This component returns the appropriate Tag OR Icon component based on the title it receives
+describe("DashbarCardTagOrIcon", () => {
+  it("Should render a 'Route' icon", () => {
+    mount(<DashbarCardTagOrIcon title={PATHWAYS} />);
+
+    // ensure the 'Route' icon is displayed
+    cy.get("[data-cy=route-icon]").should("exist");
+  });
+
+  /******************************************************/
```

Comment:
These aren't really necessary, or? :)

Reply:
You are right :) 
I will remove them 👍🏼